examples/auth0middleware/main.go

```diff
@@ -20,14 +20,14 @@ func main() {
 	// Create the keyfunc options. Use an error handler that logs.
 	options := keyfunc.Options{
 		RefreshErrorHandler: func(err error) {
-			log.Printf("There was an error with the jwt.KeyFunc\nError: %s", err.Error())
+			log.Printf("There was an error with the jwt.KeyFunc\nError:%s\n", err.Error())
```

Comment:
By the way, why did you decide to add newlines at the ends of all the example prints? I believe the `log` package automatically adds them for you and ignores any you add. 

I tested this here: https://play.golang.org/p/Eaevrr9BPn0

Reply:
Yeah, the standard logger does. But one of the ones at work doesn't. Got in the habit of adding that extra newline so I get to read log statements consistently.